android/WALT/app/src/main/res/xml/preferences.xml

```diff
@@ -1,40 +1,70 @@
-<PreferenceScreen
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.v7.preference.PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:walt="http://schemas.android.com/apk/res/org.chromium.latency.walt">
 
-    <PreferenceCategory
-        android:title="Common">
+    <android.support.v7.preference.PreferenceScreen
+        android:key="pref_screen_response_screen"
+        android:persistent="false"
+        android:title="Screen Response">
 
-        <SwitchPreferenceCompat
-            android:key="pref_verbose_log"
-            android:title="Verbose logging"
-            android:summary="(not implemented)"/>
+        <org.chromium.latency.walt.NumberPickerPreference
+            android:defaultValue="20"
+            android:dialogTitle="Number of blinks for screen latency measurement"
+            android:key="@string/preference_screen_blinks"
+            android:summary="%s blinks per test"
+            android:title="Screen latency test length"
+            walt:maxValue="100"
+            walt:minValue="1" />
+
+    </android.support.v7.preference.PreferenceScreen>
 
-    </PreferenceCategory>
+    <android.support.v7.preference.PreferenceScreen
+        android:key="pref_audio_screen"
+        android:persistent="false"
+        android:title="Audio">
 
-    <PreferenceCategory
-        android:title="Audio output">
+        <org.chromium.latency.walt.NumberPickerPreference
+            android:defaultValue="5"
+            android:dialogTitle="Number of repetitions for audio input latency"
+            android:key="@string/preference_audio_in_reps"
+            android:summary="%s repetitions per test"
+            android:title="Audio input test length"
+            walt:maxValue="50"
+            walt:minValue="1" />
 
         <org.chromium.latency.walt.NumberPickerPreference
-            android:key="pref_audio_out_reps"
-            android:title="Audio out reps"
-            android:summary="(not implemented)"
-            android:dialogTitle="Number of reps for audio out latency"/>
+            android:defaultValue="10"
+            android:dialogTitle="Number of repetitions for audio output latency"
+            android:key="@string/preference_audio_out_reps"
+            android:summary="%s repetitions per test"
+            android:title="Audio output test length"
+            walt:maxValue="50"
```

Comment:
Let's set all the maxVals to something much higher, say 1000 for all of them.
For some tests 1000 reps would mean we need to re-sync clock in the middle, but that's something we need to do anyway.